ui/src/components/Carousel.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+export default function Carousel() {
+  const languages = [
+    "Assamese",
+    "Bengali",
+    "English",
+    "Gujarati",
+    "Hindi",
+    "Kannada",
+    "Malayalam",
+    "Marathi",
+    "Odia",
+    "Punjabi",
+    "Tamil",
+    "Telgu",
+    "Urdu",
+  ];
+  return (
+    <div className="carousel carousel-center max-w-full p-4 space-x-4 bg-gray-100 rounded-box mt-12 border-4 border-primary">
```

Comment:
Small thought: Can we use some npm library to make this auto-scroll after a few seconds?

Reply:
I was thinking the same, I think the carousel can be revamped into better one so, I think we should take some time to rethink this section altogether.

Reply:
<img width="233" alt="image" src="https://github.com/kurianbenoy/Indic-Subtitler/assets/24592806/ff19f9af-e482-4821-a033-a72f4198832d">

Thanks for sharing this idea @BodaNabeel

Reply:
Alright.